### ISSUE TITLE
Can't maxsize correctly

### ISSUE THREAD

**Issue by littlewhitecloud** [littlewhitecloud/CustomTkinterTitlebar#3]
![image](https://user-images.githubusercontent.com/71159641/209413716-f0f89d99-9275-4f33-be1f-f79b37da15bb.png)
The close button only shows half. I don't know why it will happen?

**Comment by littlewhitecloud** [comment]
Fine, I fixed it, just self.w sub 20.

**Comment by littlewhitecloud** [comment]
```
	def maxsize(self, event = None):
		"Maxsize Window"
		if event and self.o_m:
			self.resize()
		else:
			self.popup.entryconfig("还原", state = "active")
			self.popup.entryconfig("最大化", state = "disabled")
			self.w_x, self.w_y = self.winfo_x(), self.winfo_y()
			self.o_m = True
			self._titlemax["image"] = self._t3_hov_img
			self._titlemax["command"] = self.resize
			w, h = self.wm_maxsize()
			self.geometry("%dx%d+0+0" % (w - 20, h - 40))
```

**Comment by littlewhitecloud** [comment]
![image](https://user-images.githubusercontent.com/71159641/209737409-b695121f-d683-4930-a029-f7eb37c3bfe5.png)
Again on my another computer...

**Comment by littlewhitecloud** [comment]
Maybe got an idea.

**Comment by littlewhitecloud** [comment]
I can use WS_MAXSIZE and the window will fullscreen and I can use wm_geometry to get window's size and put it in self.s_mw and self.s_mh

**Comment by littlewhitecloud** [comment]
And even if on anthor computer, it will still have effect!?

**Comment by littlewhitecloud** [comment]
failed

**Comment by littlewhitecloud** [comment]
WS_MAXSIZE doesn't have any effect

**Comment by littlewhitecloud** [comment]
wait wait wait, 
I used wrong long p
I shoued use anthor long p

**Comment by littlewhitecloud** [comment]
Note:
Get the taskbar size

**Comment by littlewhitecloud** [comment]
Also fixed in #40
